lib/generators/decidim/templates/invisible_captcha.rb

```diff
@@ -0,0 +1,9 @@
+InvisibleCaptcha.setup do |config|
+  config.honeypots           << 'another_fake_attribute'
```

Comment:
Why are we exposing the initializer to the generated app? Don't see why the consumer would like to override this. Let's better keep this to ourselves, the developer can always add this initializer manually.